ChangeLog

```diff
@@ -1,3 +1,7 @@
+== 2025-04-11 version 0.3.29
+
+* Support Ruby 2.7 to 3.4
```

Comment:
[suggestion] Not 100% sure, but describing the dependency changes might be helpful, e.g.

```suggestion
* Support Ruby 2.7 to 3.4
* Require td-client v1 or higher
* Add dependencies: logger, mutex_m
```

Ref https://github.com/treasure-data/td-logger-ruby/compare/v0.3.28...14ea0497e0e2157964e83d7faf0682eac65484aa

Reply:
Updated the changelogs with more detail changes! And you're right, I should've paid more attention at the changelog 🙇